src/web/browser/coreVitals/coreVitals.ts

```diff
@@ -0,0 +1,74 @@
+import { getCLS, getFID, getLCP, getFCP, getTTFB } from 'web-vitals';
+
+// Stops ts assuming json values are null
+const temp: any = null;
```

Comment:
Will this change at some point (via refactor) or will it be `temp` by design? As in, is this the initial value before it gets set?

Reply:
I tried setting a type in the JSON but couldn't so this was a quick way to solve it, I'm sure it can be refactored but it's minimal and works for now.